### PR TITLE
[ADD]: Few TEST cases for the Perfect_Cube

### PR DESCRIPTION
**Pull request by imSanko**
### Describe your change:

Added some of the tests

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Add or change doctests? -- Note: Please avoid changing both code and tests in a single pull request.
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #9943".

### CODE REVIEW

**Comment by imSanko** [comment]
@cclauss can you check the tests !

File: maths/perfect_cube.py [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+      False
+      >>> perfect_cube(125)
+      True
+      >>> perfect_cube(64)
+      True
+      >>> perfect_cube(0)
+      True
+      >>> perfect_cube(1)
+      True
+      >>> perfect_cube(3.375)
```
Comment (cclauss) [inline maths/perfect_cube.py:19]: These three tests should raise ValueError

Comment (imSanko) [reply maths/perfect_cube.py]: can you have a look now?

Comment (cclauss) [reply maths/perfect_cube.py]: Below there is a ❌ next `build` test to say that it failed.  Click on the `Details` link to the right of that line.  The test failure is at about line 807 that outputs.

Comment (imSanko) [reply maths/perfect_cube.py]: Fixed @cclauss

**Comment by imSanko** [comment]
@cclauss can you explain to me whats error have committed cause it passed all the tests?

**Comment by cclauss** [comment]
We need to see some errors raised on invalid input like:
https://github.com/TheAlgorithms/Python/blob/master/maths/special_numbers/perfect_number.py#L51-L58

There are too many open pull requests to review and just a week left in Hacktoberfest.

**Pull request closed without merging**